install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -7,6 +7,14 @@ dependencies:
     version: 0.5.0
     # repository: file://../ingress
     condition: ingress.enabled
+  - name: ingress-gateway
```

Comment:
I think there is a problem that helm does not allow -'s in the names.  I could be wrong.  Might be easier to just call this ingress_gateway or ingressgateway.  @jascott1 any thoughts?

Reply:
we have sidecar-injector

Reply:
@sdake that is in values.yaml file we had to rename sidecar-injector to sidecarInjector.

Reply:
It seems to work in the tests..

Reply:
see https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/values.md.  really good work here, btw.